affinidi-did-resolver-methods/did-peer/src/lib.rs

```diff
@@ -454,8 +456,12 @@ impl DIDMethodResolver for DIDPeer {
 
                             let mut service: Service = service.into();
                             if service_count > 0 {
```

Comment:
Could we change `service_count` => `current_service_idx` or sth similar? From the change itself I had hard time understanding why we use service count as id for each service.